docs/UML/addEvent.puml

```diff
@@ -0,0 +1,39 @@
+@startuml
+Actor -> Parser : parseAddCommand()
```

Comment:
I think you should add a `activate Parser` line since the activation bar is missing from your diagram?